target/linux/ath79/dts/ar9342_ligowawe_dlb5_propeller.dts

```diff
@@ -0,0 +1,171 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar9344.dtsi"
+
+/ {
+	model = "LigoWawe DLB 5 Propeller";
+	compatible = "ligowawe,dlb5-propeller", "qca,ar9344";
+
+	aliases {
+		serial0 = &uart;
+		led-status = &status;
+	};
+
+	keys {
+		compatible = "gpio-keys-polled";
+		poll-interval = <20>;
+		#address-cells = <1>;
+		#size-cells = <0>;
+		pinctrl-names = "default";
+                pinctrl-0 = <&jtag_disable_pins>;
+
+		reset {
+			label = "Reset button";
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio 17 GPIO_ACTIVE_LOW>;
+			debounce-interval = <60>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		status: power {
+			label = "ligowawe:green:power";
```

Comment:
According to the device tree specification it should be `<board>:<colour>:<function>`.

We are using the vendor name instead of the board name only in really rare cases where a lot of similar boards sharing a common dtsi (tp-link for example). I don't expect to see a huge amount of similar LigoWawe boards.

Reply:
Ok,I will change that.
There are more or less around 10 LigoWawe products that are really similar.